source/_docs/settings-php.md

```diff
@@ -112,27 +112,29 @@ Yes, but only if at least one other file (e.g. `settings.php`) is present within
 
 #### How can I write logic based on the Pantheon server environment?
 
-Depending on your use case, there are two possibilities:
+Depending on your use case, there are three possibilities:
 
-For web only actions, like redirects, check for the existence of $\_SERVER['PANTHEON\_ENVIRONMENT']. If it exists, it will contain a string with the current environment (Dev, Test, or Live).
+ - For web only actions, like redirects, check for the existence of `$\_SERVER['PANTHEON\_ENVIRONMENT']`. If it exists, it will contain a string with the current environment (Dev, Test, or Live):
 
-    // Pantheon - web only.
-    if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
-      // Only on dev web environment.
-      if ($_SERVER['PANTHEON_ENVIRONMENT'] == 'dev') {
-        // Custom code.
-      }
-    }
+        // Pantheon - web only.
+        if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
+          // Only on dev web environment.
+          if ($_SERVER['PANTHEON_ENVIRONMENT'] == 'dev') {
+            // Custom code.
+          }
+        }
 
-For actions that should take place on every environment, such as Redis caching, use the constant ​PANTHEON\_ENVIRONMENT. Again, it will contain Dev, Test, or Live.
+ - For actions that should take place on every environment, such as Redis caching, use the constant `PANTHEON\_ENVIRONMENT`. Again, it will contain Dev, Test, or Live:
 
-    // Pantheon - all operations.
-    if (defined('PANTHEON_ENVIRONMENT')) {
-      // Only on dev environment.
-      if (PANTHEON_ENVIRONMENT == 'dev') {
-        // Custom code.
-      }
-    }
+        // Pantheon - all operations.

```

Comment:
Suggest replacing the vague snippet with our D8 recommended config for Redis, since we reference it as an example use case:

```
// Configure Redis on all Pantheon environments for Drupal 8

if (defined('PANTHEON_ENVIRONMENT')) {
  // Include the Redis services.yml file. Adjust the path if you installed to a contrib or other subdirectory.
  $settings['container_yamls'][] = 'modules/redis/example.services.yml';

  //phpredis is built into the Pantheon application container.
  $settings['redis.connection']['interface'] = 'PhpRedis';
  // These are dynamic variables handled by Pantheon.
  $settings['redis.connection']['host']      = $_ENV['CACHE_HOST'];
  $settings['redis.connection']['port']      = $_ENV['CACHE_PORT'];
  $settings['redis.connection']['password']  = $_ENV['CACHE_PASSWORD'];

  $settings['cache']['default'] = 'cache.backend.redis'; // Use Redis as the default cache.

  // Always set the fast backend for bootstrap, discover and config, otherwise this gets lost when redis is enabled.
  $settings['cache']['bins']['bootstrap'] = 'cache.backend.chainedfast';
  $settings['cache']['bins']['discovery'] = 'cache.backend.chainedfast';
  $settings['cache']['bins']['config']    = 'cache.backend.chainedfast';
}
```